### PR TITLE
CRUD category

### PR DESCRIPTION
**Pull request by FaduaJ**

### CODE REVIEW

File: src/category/category.service.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+export class CategoryService {
+  constructor(
+    @InjectRepository(Category)
+    private categoryRepository:Repository<Category>,
+  ){
+
+  }
+  async create(createCategoryDto: CreateCategoryDto) {
+    try {
+      const {name,description}= createCategoryDto;
```
Comment (jhersON1) [inline src/category/category.service.ts]: space before "="

File: src/category/category.service.ts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  
+    return updatedCategory;
+  
+    } catch (error) {
+      
+    }
+     
+  }
+
+  remove(id: number) {
```
Comment (jhersON1) [inline src/category/category.service.ts]: if you don't have to use this function, delete it

File: src/category/entities/category.entity.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { Column, Entity, PrimaryGeneratedColumn } from "typeorm";
```
Comment (jhersON1) [inline src/category/entities/category.entity.ts]: In all files, use single quotes

File: src/category/entities/category.entity.ts [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+@Entity('Category')
+export class Category {
+    @PrimaryGeneratedColumn('uuid')
+    id: string
+
+    @Column()
+    name:string
```
Comment (jhersON1) [inline src/category/entities/category.entity.ts]: In all files put space after ":" and put ";"  in a final of the phrase

File: src/category/category.controller.ts
Comment (jhersON1) [inline src/category/category.controller.ts:1]: delete all function if you not use

**Review by jhersON1 — COMMENTED** [review]
check comments

File: src/category/category.service.ts [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+    const updatedCategory = await this.categoryRepository.save({
+      ...category,
+      ...updateCategoryDto,
+    });
+  
+    return updatedCategory;
+  
+    } catch (error) {
+
```
Comment (jhersON1) [inline src/category/category.service.ts]: is empty

**Comment by FaduaJ** [comment]
error controlled

**Review by jhersON1 — APPROVED** [review]

**Pull request merged**